src/main/java/org/example/parampammusic/config/ThymeleafConfig.java

```diff
@@ -0,0 +1,44 @@
+package org.example.parampammusic.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.thymeleaf.spring6.SpringTemplateEngine;
+import org.thymeleaf.spring6.templateresolver.SpringResourceTemplateResolver;
+import org.thymeleaf.spring6.view.ThymeleafViewResolver;
+import org.thymeleaf.templatemode.TemplateMode;
+
+@Configuration
+public class ThymeleafConfig implements WebMvcConfigurer {
+
+    @Override
+    public void addViewControllers(ViewControllerRegistry registry) {
+        registry.addViewController("/loginForm").setViewName("loginForm");
+        registry.addViewController("/registrationForm").setViewName("registration");
+    }
+
+    @Bean
+    public SpringResourceTemplateResolver templateResolver() {
```

Comment:
Do you need it for your viewvs?
You are using spring boot and can do in application file